### ISSUE TITLE
Brouter-Web on mobile: adjust route in between two waypoints

### ISSUE THREAD

**Issue by tauwael** [nrenner/brouter-web#285]
Previously Brouter-Web (mobile) had the functionality to adjust the route between two waypoints by adding a third waypoint. All you had to do was to click the route and swipe that new waypoint to wherever you want.

This seems to be omitted since the new major update. I was quite used to doing this, so it's hard for me now having to remove all waypoints up to the part I want to adjust.

A fix would be highly appreciated.

Kudos to you guys!

**Comment by nrenner** [comment]
Thanks for reporting.

I personally haven't really used it on mobile so I couldn't tell if and when this has stopped working.

My first thought was that this might be caused by changed Leaflet touch/pointer flags, but switching back to Leaflet 1.4.0 didn't seem to help. So needs further investigation. Firefox also doesn't work.

**Comment by nrenner** [comment]
I was wrong, it actually is caused by the update to Leaflet 1.6.0 in v0.11.0.

It is caused by Leaflet/Leaflet#6855 and fixed by Leaflet/Leaflet#7010, but not released yet.

The issue is that `L.Browser.pointer` is set to `false` in Chrome browsers, therefore the `Tap` handler is enabled on mobile Chrome:

https://github.com/Leaflet/Leaflet/blob/d1a1e97b8290f642eb677284af53c2db64199a76/src/map/handler/Map.Tap.js#L131-L136
```js
if (Browser.touch && !Browser.pointer) {
	Map.addInitHook('addHandler', 'tap', Tap);
}
```

The Tap handler replaces the default touch events with simulated events, but does not emit a `mouseover` event on touch. As a result, the route drag handle does appear on touch, so dragging the route to insert a waypoint is not possible.

Instead of switching back to a previous Leaflet version or somehow using master, the fix is to [disable the Tap handler](https://leafletjs.com/reference-1.6.0.html#map-tap) by setting `tap: false` in the map options (b11c182).
